tests/conftest.py

```diff
@@ -11,3 +11,24 @@
 @pytest.fixture
 def getkey():
     return eqxi.GetKey()
+
+
+# TODO: this can probably be done in a more streamlined fashion!
+# Skip benchmarks on default test runs. Request these explicitly (e.g. when stress-
+# testing new features).
```

Comment:
It looks like this can be done with a `skipif`: https://stackoverflow.com/a/43938191/22545467